client/blocks/comments/post-comment-list.jsx

```diff
@@ -210,110 +236,115 @@ class PostCommentList extends React.Component {
-						{
-							translate( 'View earlier comments (Showing %(shown)d of %(total)d)', {
-								args: {
-									shown: displayedCommentsCount,
-									total: actualTotalCommentsCount
-								}
-							} )
-						}</span> : null }
-				</div> }
+				{ ( this.props.showCommentCount || showViewEarlier ) &&
+					<div className="comments__info-bar">
+						{ }
+						{ this.props.showCommentCount && <CommentCount count={ actualTotalCommentsCount } /> }
+						{ showViewEarlier
+							? <span
+									className="comments__view-earlier"
+									onClick={ this.viewEarlierCommentsHandler }
+								>
+									{ translate( 'View earlier comments (Showing %(shown)d of %(total)d)', {
+										args: {
+											shown: displayedCommentsCount,
+											total: actualTotalCommentsCount,
+										},
+									} ) }
+								</span>
+							: null }
+					</div> }
 				{ showFilters &&
 					<SegmentedControl compact primary>
 						<SegmentedControlItem
 							selected={ commentsFilter === 'all' }
-							onClick={ this.handleFilterClick( 'all' ) }>{ translate( 'All' ) }
+							onClick={ this.handleFilterClick( 'all' ) }
+						>
+							{ translate( 'All' ) }
 						</SegmentedControlItem>
 						<SegmentedControlItem
 							selected={ commentsFilter === 'approved' }
-							onClick={ this.handleFilterClick( 'approved' ) }>{ translate( 'Approved', { context: 'comment status' } ) }
+							onClick={ this.handleFilterClick( 'approved' ) }
+						>
+							{ translate( 'Approved', { context: 'comment status' } ) }
 						</SegmentedControlItem>
 						<SegmentedControlItem
 							selected={ commentsFilter === 'unapproved' }
-							onClick={ this.handleFilterClick( 'unapproved' ) }>{ translate( 'Pending', { context: 'comment status' } ) }
+							onClick={ this.handleFilterClick( 'unapproved' ) }
+						>
+							{ translate( 'Pending', { context: 'comment status' } ) }
 						</SegmentedControlItem>
 						<SegmentedControlItem
 							selected={ commentsFilter === 'spam' }
-							onClick={ this.handleFilterClick( 'spam' ) }>{ translate( 'Spam', { context: 'comment status' } ) }
+							onClick={ this.handleFilterClick( 'spam' ) }
+						>
+							{ translate( 'Spam', { context: 'comment status' } ) }
 						</SegmentedControlItem>
 						<SegmentedControlItem
 							selected={ commentsFilter === 'trash' }
-							onClick={ this.handleFilterClick( 'trash' ) }>{ translate( 'Trash', { context: 'comment status' } ) }
+							onClick={ this.handleFilterClick( 'trash' ) }
+						>
+							{ translate( 'Trash', { context: 'comment status' } ) }
 						</SegmentedControlItem>
-					</SegmentedControl>
-				}
+					</SegmentedControl> }
 				{ this.renderCommentsList( displayedComments ) }
+				{ showViewLater &&
+					<span
+						className="comments__view-earlier"
+						onClick={ this.viewEarlierCommentsHandler }
+					>
+						{ translate( 'View later comments (Showing %(shown)d of %(total)d)', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'View earlier comments (Showing %(shown)d of %(total)d)' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).